ethereum/stratum/src/main/java/org/hyperledger/besu/ethereum/stratum/StratumServer.java

```diff
@@ -39,20 +45,26 @@
 public class StratumServer implements PoWObserver {
 
   private static final Logger logger = getLogger();
+  private static final UInt256 DIFFICULTY_1_TARGET = UInt256.valueOf(2).pow(256);
 
   private final Vertx vertx;
   private final int port;
   private final String networkInterface;
   private final AtomicBoolean started = new AtomicBoolean(false);
+  private final AtomicLong numberOfMiners = new AtomicLong(0);
+  private final AtomicDouble difficultyGauge = new AtomicDouble(0.0);
```

Comment:
It's not actually a gauge, we get the gauge from `metricsSystem.createGauge`.  Perhaps `currentDifficulty`?

Reply:
alright

Reply:
done